main.js

```diff
@@ -211,22 +212,32 @@ function loadFile(file) {
 
 //Read the config file
 var readConfigFile = async function () {
-  var x = document.getElementById("configFile");
-  var input = "";
-
-  if ("files" in x) {
-    if (x.files.length == 0) {
-      var morphoLabel = document.getElementById("labelspMorphoFeats");
-      morphoLabel.style.visibility = "hidden";
-      txt = "Select config file.";
-    } else {
-      var file = x.files[0];
-      if (!alreadyReadConfigFiles.includes(file)) {
-        alreadyReadConfigFiles.push(file);
-        await loadFile(file);
+  var selectList = document.getElementById("config_file_selector");
+  var file;
+  // default config file
+  if (selectList.selectedIndex > 0 && selectList.selectedIndex < selectList.options.length - 1) {
+    file = defaultConfigFiles[selectList.selectedIndex-1];
+  }
+  else if (selectList.selectedIndex == selectList.options.length - 1){ // user uploaded config file
+    var x = document.getElementById("configFile");
+    var input = "";
+    if ("files" in x) {
+      if (x.files.length == 0) {
+        var morphoLabel = document.getElementById("labelspMorphoFeats");
+        morphoLabel.style.visibility = "hidden";
+        txt = "Select config file.";
+        return;
+      } else {
+        file = x.files[0];
       }
     }
   }
+  else return; // no config file selected
+  
+  if (!alreadyReadConfigFiles.includes(file)) {
```

Comment:
can you add a comment explaining what this does?

Reply:
The condition `if (!alreadyReadConfigFiles.includes(file)){}` is from the original code. I guess the purpose is to only read each config file once. However, this does not seem necessary at the moment since we can only read maximum of 1 config file in the upload screen before going to the edit screen. When we want to go back to upload screen again to upload another config file, the window reloads and `alreadyReadConfigFiles` array is reset to empty.